lib/cli/interactive-setup/service.js

```diff
@@ -127,13 +140,43 @@ module.exports = {
       }
       projectName = await resolveProjectNameInput(context.options, workingDir);
       projectDir = join(workingDir, projectName);
-      await createFromTemplate(projectType, projectDir);
+      const templateUrl = `https://github.com/serverless/examples/tree/master/${projectType}`;
+      try {
+        process.stdout.write(`\n${chalk.green(`Downloading "${projectType}" template...`)}\n`);
+        await downloadTemplateFromRepo(templateUrl, projectType, projectName, true);
+      } catch (err) {
+        throw new ServerlessError(
+          'Could not download template. Ensure that you are using the latest version of Serverless Framework.',
+          'TEMPLATE_DOWNLOAD_FAILED'
+        );
+      }
+    }
+
+    let hasPackageJson = false;
+    try {
+      await fsp.access(join(projectDir, 'package.json'));
+      hasPackageJson = true;
+    } catch {
+      // pass
+    }
+
+    if (hasPackageJson) {
+      process.stdout.write(
+        `\n${chalk.green(`Installing dependencies with npm in '${projectName}' folder.`)}\n`
+      );
+      await spawn('npm', ['install'], { cwd: projectDir });
```

Comment:
Here we should eventually support internally bundled `npm` (e.g. we do that with `plugin install`, also in case user not having _npm_ installed (which we can detect afaik via `ENOENT` code), we may show friendly message: _Cannot install npm dependendies as "npm" is not found to be installed. Please install npm and run "npm install" in folder of created service_